back-office/src/views/HomeView.vue

```diff
@@ -35,13 +70,38 @@
 import { TOAST_TYPE } from '@/enums';
 import type { User } from '@/models';
 import { UserService } from '@/services';
-import { useToastStore, useUserStore } from '@/stores';
+import { useTagStore, useToastStore, useUserStore } from '@/stores';
 import type { EventSourcePolyfill } from 'event-source-polyfill';
 import { computed, onBeforeUnmount, ref } from 'vue';
 
 const userStore = useUserStore();
 const toastStore = useToastStore();
+const tagStore = useTagStore();
+
+const user = userStore.getContextUser;
+let {...commentaire} = ref('')
```

Comment:
Pourquoi pas commentaire = ref('')

Reply:
meme pb que sur le challenge, ça modifie le commentaires des autres tags dans le v-for